packages/tasit-contracts/src/helpers/removeFunctionsFromABI.js

```diff
@@ -31,6 +32,7 @@ const removeFromEstateRegistry = () => {
   removeFunctionsFromABI(filePath, removalCondition);
 };
 
+// eslint-disable-next-line no-unused-vars
 const removeFromMarketplace = () => {
```

Comment:
What's the reason this is unused right now?

Reply:
There is a comment on top of the file:

```javascript
// Remove overloaded functions
// This helper isn't be used right now, but it'll be useful on that issue:
// https://github.com/tasitlabs/TasitSDK/issues/241 (step 3)
```

Reply:
Ah ok - so when you said "this helper", you meant the file, not just one function?

Reply:
Yep.

Reply:
Cool, thanks. Resolving this